refactor: new structure android (#707)
ZupIT/beagle-docs: content/en/android/customization/widgets/_index.md

---
title: Custom Widget
weight: 129
description: >-
  You will find here an example on how to create a component and a customized
  widget
---

---

## Introduction

> Beagle already has **basic widgets** you can use to create interface components of your server-driven application. 

Reviewer: ```suggestion
Beagle already has **basic widgets** used to create interface components of your server-driven application. 
```

> However, your application may need more customized components \(Custom Views\), and to make them 'visible' to Beagle, you have to create a customized widget. You can create many new components you want, since it **makes your application's Views, 'visible' to Beagle**. 

Reviewer: ```suggestion
However, your application may need more customized components \(Custom Views\), and to make them 'visible' to Beagle, you have to create a customized widget. Because of that, you can create custom widgets that will be visible / rendered on Beagle
```
